bin/appup_test.sh

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env bash
+set -euo pipefail
+BASEDIR=$(dirname $(realpath "$0"))
+
+## @DOC
+## The script is used to test the appup.src file with following steps,
+## 1) It builds all release tar files of *current* vsn and old vsns that they are defined in ${supported_rel_vsns}
+## 2) It untar all the tar files to one tmp dir then build the *relup* file from there. see make_relup()
+## 3) It then append the relup file to the tar file of current release, see make_relup()
+## 4) It then test the old vsn upgrade and downgrade, see test_relup()
+
+# GLOBAL
+app=emqtt
+supported_rel_vsns="1.4.6"
+
+die()
+{
+    echo "$1"
+    exit 1;
+}
+
+build_and_save_tar() {
+    dest_dir="$1"
+
+    if rebar3 plugins list |grep relup_helper; then
+        # for old vsn that do not have relup_helper
+        rebar3 as emqtt_relup_test do relup_helper gen_appups,tar;
```

Comment:
Maybe the `if` branches are switched, since this one is for the case where there's no `relup_helper`?